Makefile

```diff
@@ -342,20 +342,22 @@ cleanup_coordinator:
 containers: gen_grpc						## Build Docker containers for Minikube
 	minikube image build -f ./api/Dockerfile . -t local/api-server:stable & \
 	minikube image build -f ./dashboard/Dockerfile . -t local/dashboard:stable & \
-	minikube image build -f ./coordinator/Dockerfile . -t local/coordinator:stable & \
+	minikube image build -f ./coordinator/Dockerfile.old --build-opt=build-arg=TESTING=True . -t local/coordinator:stable & \
```

Comment:
This will lead to a faster build because it would only install `Python3.7` instead of `3.7-3.10`. And the `.old` version is the version we use.